Python.gitignore

```diff
@@ -92,3 +92,6 @@ ENV/
 
 # Rope project settings
 .ropeproject
+
+# mkdocs documentation
+site/
```

Comment:
If this is always at the root of the repository it can be written as `/site` - this will help prevent clashes with other folders named the same...